### DISCUSSION TITLE
Show indent lines within code blocks

### DISCUSSION THREAD

**Discussion by datkumar** [expressive-code/expressive-code#378] [Feature Requests]
Maybe I'm nitpicking but it would be nice to show indent lines for the code blocks as it's easier for the eyes to follow:

VSCode:
![Screenshot From 2024-11-15 22-59-40](https://github.com/user-attachments/assets/f83a381d-6385-42a2-8e5c-03afc43ddf44)

Expressive-code block on Astro site:
![Screenshot From 2024-11-15 23-00-50](https://github.com/user-attachments/assets/1820317c-5809-4366-b28d-c61d400258ff)

**Comment by delucis** [comment]
Hello! We are moving feature requests and support questions to discussions and using issues only for bugs to help better track those going forward. See you in the [Discussions](https://github.com/expressive-code/expressive-code/discussions) tab!
